pyaerocom/aeroval/modelmaps_engine.py

```diff
@@ -122,7 +123,12 @@ def _process_map_var(self, model_name, var, reanalyse_existing):
         """
         data = self.read_model_data(model_name, var)
         check_var_ranges_avail(data, var)
-        varinfo = VarinfoWeb(var)
+
+        if var in var_ranges_defaults.keys():
```

Comment:
the `keys` method is not needed to check if a particular key is in a dictionary.
You could write something like

``` python
        if var in var_ranges_defaults:
```

Reply:
noted